src/decode.h

```diff
@@ -580,6 +580,10 @@ typedef struct Packet_
     uint8_t *payload;
     uint16_t payload_len;
 
+#ifdef HAVE_NDPI
+    uint16_t ip_len;
```

Comment:
I guess we can avoid to store this, and recompute it based on the total packet length and offset of ip header...

Reply:
Yeah, we'll want to get rid of additional #ifdef here as well for the move to a plugin.